### ISSUE TITLE
Import issues continue

### ISSUE THREAD

**Issue by seruum** [jangko/nimpdf#17]
I was about to open a ticket explaining the import issues I was having when I saw that some updates had already been made in regard to that. However, they still exist for me. I'm using 0.19.0, which I thought might be part of the problem, but I'm not certain.

If it helps, to get it to work, I moved all the nimPDF files into the first directory in nimble (nimpdf-0.4.2 I think) and then had to download the zip file and manually add the "private" directory to nimPNG. This looks like it may have been addressed already.

When I saw the new changes, I removed nimPDF using nimble and then re-installed it. I didn't go further after that because of time, but it looked to act the same as before in that "nimpdf could not be found".

FWIW, I'm and hobbyist noob, so I could be missing something.
Thank you

**Comment by jangko** [comment]
well, I think you should `import nimPDF/nimPDF`, and not only `import nimPDF`.
artefact from old days

**Comment by seruum** [comment]
That worked.
I was trying to build/run the pdf_encoding_list.nim example. When I made the changes you mentioned it was able to compile. It ran fine in release mode, but in debug mode I got the following response.

pdf_encoding_list.nim(103) pdf_encoding_list
pdf_encoding_list.nim(95) main
nimPDF.nim(113)          writePDF
page.nim(398)            writePDF
page.nim(367)            putCatalog
page.nim(318)            putOutlines
page.nim(288)            putOutlineItem
page.nim(165)            putDestination
objects.nim(443)         addElement
system.nim(3790)         failedAssertImpl
system.nim(3783)         raiseAssert
system.nim(2830)         sysFatal
Error: unhandled exception: C:\Users\me\ .nimble\pkgs\nimPDF-0.4.2\nimpdf\objects.nim(443, 9) `(val.objID and 0x80000000'i32) == 0`  [AssertionError]
Error: execution of an external program failed: 'c:\Users\me\Projects\NimTest\pdf_encoding_list.exe '
The terminal process terminated with exit code: 1

Thanks again
